themes/bootstrap3/templates/record/explain.phtml

```diff
@@ -0,0 +1,13 @@
+<?php if ($this->disabled): ?>
+  <h3><?= $this->transEsc('explain_disabled', ['%%searchClassId%%' => $this->driver->getSourceIdentifier()]) ?></h3>
```

Comment:
Very minor style suggestion: we don't usually include a space after `<?=`.

Reply:
Fixed. In my opinion spaces improve the readability. But maybe I'm just more used to it.